
Rediffed for sysvinit-2.86, altugh it is a question why we actually patch it
away ...

  - Rene Rebe <rene@exactcode.de>

--- sysvinit-2.86/src/Makefile	2004-06-09 14:47:45.000000000 +0200
+++ sysvinit-2.86-fixed/src/Makefile	2004-12-11 18:18:10.228142800 +0100
@@ -24,11 +24,11 @@
 MAN8	+= shutdown.8 telinit.8
 
 ifeq ($(DISTRO),)
-BIN	+= mountpoint
-SBIN	+= sulogin bootlogd
-USRBIN	+= utmpdump wall
-MAN1	+= mountpoint.1 wall.1
-MAN8	+= sulogin.8 bootlogd.8
+BIN	+= 
+SBIN	+= sulogin
+USRBIN	+= wall
+MAN1	+= wall.1
+MAN8	+= sulogin.8
 endif
 
 ifeq ($(DISTRO),Debian)
@@ -71,9 +71,6 @@
 mountpoint:	mountpoint.o
 		$(CC) $(LDFLAGS) -o $@ mountpoint.o
 
-utmpdump:	utmpdump.o
-		$(CC) $(LDFLAGS) -o $@ utmpdump.o
-
 runlevel:	runlevel.o
 		$(CC) $(LDFLAGS) -o $@ runlevel.o
 
